src/HttpClient.php

```diff
@@ -34,7 +34,7 @@ class HttpClient
      *
      * @var int
      */
-    const MAX_SIZE = 500000;
+    const MAX_SIZE = 1048576;
```

Comment:
This could instead be the calculation for the value, so its obvious at first glance what is happening here.

```suggestion
    const MAX_SIZE = 1024 * 1024;
```

Reply:
PHP 5.5 does not seem to like that I use an expression instead of a constant value. https://travis-ci.com/bugsnag/bugsnag-php/jobs/227394423

I'll need to keep it as a whole number for as long as we support that PHP version in the CI.

Reply:
Yeh, only 5.6+ allow a wider class of expessions as constants, unfortunately.

Reply:
I’d forgotten that it might anger The Ancient Ones. Well maybe add a comment to say it’s a Mb.